plugins/vertica/database_credentials.go

```diff
@@ -0,0 +1,55 @@
+package vertica
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DatabaseCredentials,
+		DocsURL:       sdk.URL("https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/ConnectingToVertica/vsql/Install/InstallingTheVsqlClient.htm"), 
+		ManagementURL: sdk.URL("https://www.vertica.com/try/"), 
```

Comment:
This URL should be something user-specific, where the user can manage their account.

If there is no such URL, we should exclude this field.

Reply:
there is now such default url